Dockerfile

```diff
@@ -0,0 +1,9 @@
+# https://docs.docker.com/compose/rails/
+FROM ruby:2.2.0
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
After updating and installing you can remove all unnecessary files so this layer is thinner.

```
&& apt-get clean \
&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```